gulpfile.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+// This file is only for generating the docs
+// No need to use any of this if working on the main website
+
+const { src, dest, series } = require('gulp');
+const pump = require('pump');
+const through2 = require('through2');
+const frontMatter = require('gray-matter');
+const download = require('github-download-directory');
+
+// Exports for task registration
+exports.default = series(fetchDocs, removeComments);
+
+const owner = 'gulpjs';
+const repo = 'gulp';
+const directory = 'docs';
+
+const fmOptions = {
+  delimiters: ['<!-- front-matter', '-->']
+};
+
+function fetchDocs() {
```

Comment:
@phated Question Part 1: So here is my question. I see where you are fetching the docs from `wip-docs`. But they come through from Github with what appears to be no front-matter like I added on all the files. Perhaps that's a bug?

Reply:
Looks like there's a bug.  Looking into it.